master/static/srv/notebook-entrypoint.sh

```diff
@@ -41,7 +42,7 @@ test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
 "$DET_PYTHON_EXECUTABLE" /run/determined/jupyter/check_idle.py &
 
 READINESS_REGEX='^.*Jupyter Server .* is running.*$'
-exec jupyter lab --ServerApp.port=${NOTEBOOK_PORT} \
+jupyter lab --ServerApp.port=${NOTEBOOK_PORT} \
```

Comment:
Move the following lines left too, I guess. I mean, actually we should just start using `shfmt`.